tools/test_containers_compose

```diff
@@ -1,12 +1,15 @@
 #!/bin/bash
 
 set -euo pipefail
+#https://github.com/os-autoinst/openQA/pull/3864/checks?check_run_id=2440471511
+#https://github.com/os-autoinst/openQA/pull/3864/checks?check_run_id=2444792522
 
 for i in webui worker; do
   (
     cd container/$i/ &&
      for retry in {2..0}; do sudo docker-compose build && break; echo "Remaining retries $retry"; done &&
-    sudo docker-compose up -d &&
-    (docker-compose ps --services --filter status=stopped | grep "^[[:space:]]*$") || (docker-compose logs; sudo docker-compose ps; exit 1)
+    (sudo docker-compose up -d || docker-compose logs; docker-compose ps; exit 1) &&
+    (docker-compose ps --services --filter status=stopped | grep "^[[:space:]]*$") || (docker-compose logs; sudo docker-compose ps; exit 1) &&
+    (sleep 1 && docker-compose ps | tail -n +3 | grep unhealthy && docker-compose logs && sudo docker-compose ps && exit 1)
```

Comment:
Sometimes `docker-compose ps` is called with `sudo` and sometimes without. I assume you don't need it in all cases.

Reply:
Yes, but I think will do in an other PR